nanoFramework.System.Net.Http/Http/System.Net._OutputNetworkStreamWrapper.cs

```diff
@@ -139,13 +139,22 @@ public override void Close()
         /// </summary>
         public override void Flush()
         {
-            if (m_headersSend != null)
+            // skigrinder - 2020-12-10 added exception handling
+            try
+            {
+                if (m_headersSend != null)
+                {
+                    // Calls HttpListenerResponse.SendHeaders. HttpListenerResponse.SendHeaders sets m_headersSend to null.
+                    m_headersSend();
+                }
+                if (m_Stream != null)
+                {
+                    m_Stream.Flush();
+                }
+            }
+            catch
             {
```

Comment:
Need a comment here explaining why we can safely use a catch all.

Reply:
Done.